api/__init__.py

```diff
@@ -0,0 +1,54 @@
+import base64
+
+from flask import Flask, request
+import numpy as np
+
+
+def create_app():
+    app = Flask(__name__)
+
+    @app.route('/')
+    def routes():
+        return {
+            "spectrum": "/spectrum",
+            "mean": "/mean",
+            "median": "/median",
+        }
+
+    @app.route('/spectrum', methods=['POST', 'GET'])
+    def spectrum():
+        if request.method == 'GET':
+            return {'params': ['sampling_rate', 'units', 'dtype']}
+
+        req = request.get_json()
+        sr = float(req['sampling_rate'])
+        units = req["units"]
+        dtype = req['dtype'] if 'dtype' in req else "float64"
+        spectrum = np.frombuffer(base64.b64decode(req["spectrum"].encode('utf-8')), dtype=dtype).tolist()
+        samples_per_signal = len(spectrum)
+        duration = samples_per_signal / sr
+        return {'spectrum': spectrum}
```

Comment:
Not much has happened in this endpoint.  Why have units, samples_per_signal, and duration defined if they are not going to be used.  What is ````/spectrum```` attempting to compute from the time series here?

Reply:
#5